Docker/test/selenium/specs/elasticsearch/elasticsearch.js

```diff
@@ -44,7 +44,7 @@ describe( 'ElasticSearch', function () {
 		assert( alias === itemAlias );
 	} );
 
-	it( 'Should able able to run UpdateSearchIndexConfig.php', function () {
+	it.skip( 'Should able able to run UpdateSearchIndexConfig.php', function () {
```

Comment:
What's the plan for this? Can they be re-enabled in some other form in a follow up (maybe as an assert that it actually worked)?

Reply:
Hmm I'm not sure i had a hard time deleting them because I spent so much time on it but lets just get rid of them?

Reply:
https://github.com/wmde/wikibase-release-pipeline/pull/124